bazel/external/proxy_wasm_cpp_host.BUILD

```diff
@@ -0,0 +1,54 @@
+licenses(["notice"])  # Apache 2
```

Comment:
Sorry, let me fix that.

Reply:
Checked.  No they can not.  The BUILD files are specific to Envoy.  The native build files have generic tests and do not have implicit dependencies on the Envoy tree..  These upstream repos are Envoy independent.  Bazel has issues such that even it injects all the dependencies from envoy into the subproject so that any BUILD file in the upstream repo must be Envoy dependent.  Consequently we need to inject the Envoy BUILD files here.